scripts/cplane/README.md

```diff
@@ -0,0 +1,54 @@
+# Vagrant Kubernetes Cluster
```

Comment:
Documentation about the script supporting work on the Control Plane

This README is created to look the same as the other ones in the project - consistency